src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -454,45 +486,27 @@ function WorkspaceMembersPage(props) {
-                        <Button
-                            medium
-                            success
-                            text={props.translate('common.invite')}
-                            onPress={inviteUser}
-                        />
-                        <Button
-                            medium
-                            danger
-                            style={[styles.ml2]}
-                            isDisabled={selectedEmployees.length === 0}
-                            text={props.translate('common.remove')}
-                            onPress={askForConfirmationToRemove}
-                        />
-                    </View>
-                    <View style={[styles.w100, styles.mt4, styles.flex1]}>
-                        <SelectionList
-                            canSelectMultiple
-                            sections={[{data, indexOffset: 0, isDisabled: false}]}
-                            textInputLabel={props.translate('optionsSelector.findMember')}
-                            textInputValue={searchValue}
-                            onChangeText={(value) => {
-                                SearchInputManager.searchInput = value;
-                                setSearchValue(value);
-                            }}
-                            disableKeyboardShortcuts={removeMembersConfirmModalVisible}
-                            headerMessage={getHeaderMessage()}
-                            headerContent={getHeaderContent()}
-                            onSelectRow={(item) => toggleUser(item.accountID)}
-                            onSelectAll={() => toggleAllUsers(data)}
-                            onDismissError={dismissError}
-                            showLoadingPlaceholder={!isOfflineAndNoMemberDataAvailable && (!OptionsListUtils.isPersonalDetailsReady(props.personalDetails) || _.isEmpty(props.policyMembers))}
-                            showScrollIndicator
-                            shouldPreventDefaultFocusOnSelectRow={!DeviceCapabilities.canUseTouchScreen()}
-                            inputRef={textInputRef}
-                        />
-                    </View>
+                <View style={[styles.w100, styles.flex1]}>
+                    <SelectionList
+                        canSelectMultiple
+                        sections={[{data, indexOffset: 0, isDisabled: false}]}
+                        textInputLabel={props.translate('optionsSelector.findMember')}
+                        textInputValue={searchValue}
+                        onChangeText={(value) => {
+                            SearchInputManager.searchInput = value;
+                            setSearchValue(value);
+                        }}
+                        disableKeyboardShortcuts={removeMembersConfirmModalVisible}
+                        headerMessage={getHeaderMessage()}
+                        headerContent={getHeaderContent()}
```

Comment:
This naming looks more common and habitual.
```suggestion
                        headerMessage={renderHeaderMessage()}
                        headerContent={renderHeaderContent()}
```

Reply:
addressed